assets/messages.yml

```
@@ -590,6 +590,23 @@ messages:
 
         %quick_links%
       fillname: []
+  not-reopen-ar:
+    - project:
+        - bds
+        - mc
+        - mcapi
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Not reopen AR ticket
+      message: |-
+        This report is currently missing crucial information. Please take a look at other comments to find out what we are looking for.
```

```suggestion
        This report is currently missing crucial information. Please take a look at the other comments to find out what we are looking for.
```
Just "other comments" sounds like any other comments, possible even on other reports.